logfire/_internal/integrations/httpx.py

```diff
@@ -44,14 +97,105 @@ def instrument_httpx(
         'meter_provider': logfire_instance.config.get_meter_provider(),
         **kwargs,
     }
-    del kwargs  # make sure only final_kwargs is used
```

Comment:
Looks like this should be restored

Reply:
I didn't remove it tho, I moved up 🤔

Reply:
you moved it down

Reply:
Ah, I think your first commit moved it down, and then I thought I moved up.

I've readded the line where it was before.